test/orange/orangeinp/detail/PolygonUtils.test.cc

```diff
@@ -87,7 +86,8 @@ TEST(PolygonUtilsTest, convex)
     VecReal2 oct{8};
     for (size_type i = 0; i < 8; ++i)
     {
-        oct[i] = {std::cos(2 * pi * i / 8), std::sin(2 * pi * i / 8)};
+        oct[i] = {std::cos(static_cast<real_type>(2 * constants::pi * i / 8)),
```

Comment:
This is an interesting pattern (since it is a test the actual usage here does not matter).  In this case do we want:
```
std::cos(static_cast<real_type>(2 * constants::pi * i / 8));
```
Which has a run-time double multiplication (by `i`)
or
```
std::cos(static_cast<real_type>(2 * constants::pi) * i / 8);
```
which the `2*` in double precision at compile time but both `* i / 8` in (potentially) single precision at run-time
or
```
std::cos(static_cast<real_type>(2 * constants::pi / 8) * i);
```
with only the `* i` in (potentially) single precision at run-time but a different ordering of the operations?